packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -164,6 +164,28 @@ export const customContentGuard = z.record(z.string());
 
 export type CustomContent = z.infer<typeof customContentGuard>;
 
+export enum MfaFactor {
+  TOTP = 'TOTP',
+  WebAuthn = 'WebAuthn',
+  BackupCode = 'BackupCode',
+}
+
+export const mfaFactorsGuard = z.nativeEnum(MfaFactor).array();
```

Comment:
Can we set multiple MAPFactors of the same type? e.g.  Two backup codes or three TOTP?

Reply:
The user can have multiple factors, but here is for feature control, so multiple with same type makes no sense.

Updated with a new check.